### PR TITLE
8 post a new comment to an existing review

### PR DESCRIPTION
**Pull request by rovercode1**

### CODE REVIEW

File: src/Components/ArticleList.jsx [hunk 1]
```diff
@@ -1,43 +1,59 @@
...
         </div>
         <div className="review-footer">
-          <p>{review.votes} Votes</p>
+          <button
+            className="default-vote"
+            id={review.votes}
+          >
+            {review.votes} Votes
+          </button>
           <p>Comments</p>
```
Comment (Slyose) [inline src/Components/ArticleList.jsx]: This doesn't actually do or show anything yet. Did you mean to show the comment count here?

**Review by Slyose — COMMENTED** [review]
Great stuff so far! Two things to consider 
- Your comments p tag on the list reviews page isn't being used for anything at the moment
- The user should be given feedback on unsuccessful post (one approach to this is having a state that keeps track of if an error has occured, the conditionally render a <p tag saying that the comment wasn't posted)

File: src/Components/SingleReview.jsx [hunk 2]
```diff
@@ -14,8 +15,9 @@ export default function SingleReview({ isLoading, setIsLoading }) {
     });
   }, []);
 
+
   const displaySingleReview = (review) => {
-    const postedAt = review.created_at.toString()
+    const postedAt = review.created_at
```
Comment (HannahBurks) [inline src/Components/SingleReview.jsx]: We never use this variable - I would either delete it or use it. The same goes for any states we don't use

File: src/Components/PostComment.jsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+import { useParams } from "react-router-dom";
+import { optimisticPostedComment } from "../handle";
+export default function PostComment({ comments, setComments}){
+  const { review_id } = useParams();
+  const [postedComment, setPostedComment] = useState('')
+  const [currentUser, setCurrentUser] = useState('grumpy19')
+  const [postedErr, setPostErr] = useState(false)
+
+  const handleSubmit = (e)=>{
+    e.preventDefault();
```
Comment (HannahBurks) [inline src/Components/PostComment.jsx:13]: When posting I can click submit multiple times and it posts duplicates - we need to add some logic to disable it whilst it is posting

File: src/Components/ArticleList.jsx [hunk 4]
```diff
@@ -1,43 +1,61 @@
...
+    setIsLoading(true);
+    fetchAllReviews().then((reviews) => {
+      setReviews(reviews);
+      setIsLoading(false);
+    });
+  }, [setIsLoading]);
 
-  const displayReviews = (review)=>{
+  const displayReviews = (review) => {
+    console.log(review)
```
Comment (HannahBurks) [inline src/Components/ArticleList.jsx]: remove any console logs such as this :)

**Review by HannahBurks — CHANGES_REQUESTED** [review]
A nice start Dean 😄 
One other thing I would note, is that it isn't very responsive - it should be a mobile first design- then use something like media queries to make it adaptable. For instance, the photo on individual reviews is too big in mobile view, it stretches.

A few changes to be made (as per my comments on the code) Once done, please send another PR 😄

**Review by HannahBurks — APPROVED** [review]
Nice one Dean 👏🏼 Merge away!

**Pull request merged**